src/app-theme.scss

```diff
@@ -71,6 +71,33 @@ $mat-app-accent: (
         A700 : #000000,
     )
 );
+
+$app-primary: mat-palette($mat-app-primary);
+$app-accent:  mat-palette($mat-app-accent);
+
+$app-theme: mat-light-theme($app-primary, $app-accent);
+
+@mixin mix-app-theme($app-theme) {
+  $primary: map-get($app-theme, primary);
+
+  $primary: map-get($app-theme, primary);
+
+  .mat-button-toggle-checked {
```

Comment:
I think we only need this part. Only for mat-checkbox instead of mat-button-toggle. We don't need to change anything else into the app-theme file.

Reply:
Thanks Alex.  =)

I have followed your suggestions. I have simplified the code as well as targeted mat-checkbox. I think it is working now.

Reply:
On github i pressed edit on the uploaded file and copy-pasted the revised code. 
Hope that is ok